bats/tests/k8s/spinkube-npm.bats

```diff
@@ -57,6 +57,10 @@ get_host() {
 }
 
 # TODO replace ingress with port-forwarding
```

Comment:
I know the todo was not added as part of this PR however, it is vague and is it still needed? or perhaps, maybe a link to an issue would be helpful.

Reply:
I just think adding an ingress controller is overkill (and a lot of source lines to review) when a simple port-forward is enough to check the pod. It simplifies the test.

The only reason I used Traefik is because the port forwarding REST API has various issues. I thought I filed a new GitHub issue about it, but can't find it right now. Will file one later.